indra/llrender/llrender.cpp

```diff
@@ -1619,7 +1619,7 @@ void LLRender::flush()
 
 		mCount = 0;
 
-        if (mBuffer)
+        if (mBuffer && LLGLSLShader::sCurBoundShaderPtr)
```

Comment:
Wouldn't this just cause a crash on else? `LL_ERRS() << "A flush call from outside main rendering thread"`

Reply:
Agree, we can just use zero mask in this case